### PR TITLE
Rename word to MAU and add link to settings docs

### PR DESCRIPTION
**Pull request by planger**
#### What it does
Renames the term *word* to *MAU* as discussed in #100 for the user-facing UI and in the code. In the options widget we also added a link to the soon-to-be existing documentation contributed in #109 and add hovers above the use of the term *MAU*.

#### How to test
As this touches several files, we should make sure the functionality isn't impacted at all.
Also it makes sense to do another search for the term *word* and make sure we caught all occurrences.

#### Review checklist

- [X] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Review by jreineckearm — APPROVED** [review]
Thanks, @planger ! Changes look good. And adding the question button to open the future "Memory Format" section in the README should be a good but not too strong way to allow users to find out about the `MAU`.
@colin-grant-work , are you happy with this change as well? Then we can drive to merge, probably after merging #108 if that makes resolving merge conflicts easier than the other way around.

File: src/webview/columns/data-column.tsx [hunk 1]
```diff
@@ -41,30 +41,30 @@ export class DataColumn implements ColumnContribution {
 
     protected renderGroups(range: BigIntMemoryRange, memory: Memory, options: TableRenderOptions): React.ReactNode {
         const groups = [];
-        let words: React.ReactNode[] = [];
+        let maus: React.ReactNode[] = [];
```
Comment (colin-grant-work) [inline src/webview/columns/data-column.tsx:44]: 'Mau' made me think of a cat noise, but here we have a Maus. :-)

**Comment by colin-grant-work** [comment]
Changes look good to me, but I merged #108 before I saw this, so now there are conflicts in the data column.

File: src/webview/columns/data-column.tsx (hunk 1 above)
Comment (planger) [reply src/webview/columns/data-column.tsx:44]: Yeah, I know that feeling. In German it is even worse, cats say "Miau" (close), but a "Maus" is a mouse, literally. :-)

Comment (jreineckearm) [reply src/webview/columns/data-column.tsx:44]: Indeed we had a discussion if to go for camel-case or not. ;-)

Comment (jreineckearm) [reply src/webview/columns/data-column.tsx:44]: And I think there is a `BigMaus` somewhere as well. :-)

**Comment by planger** [comment]
No problem, I merged the updated `main` branch and resolved the conflicts.

**Pull request merged**